Formula/clickhouse-cpp.rb

```diff
@@ -0,0 +1,77 @@
+class ClickhouseCpp < Formula
+  desc "C++ client library for ClickHouse"
+  homepage "https://github.com/ClickHouse/clickhouse-cpp#readme"
+  url "https://github.com/ClickHouse/clickhouse-cpp/archive/refs/tags/1.5.0.tar.gz"
+  sha256 "bb6f268f9c788deb9beccb0b05c2caccf77b141afa408343e09993f12bff55a9"
+  license "Apache-2.0"
+  head "https://github.com/ClickHouse/clickhouse-cpp.git", branch: "master"
+
+  depends_on "cmake" => [:build, :test]
+  depends_on "abseil"
+
```

Comment:
```suggestion

  on_linux do
    depends_on "gcc"
  end
  
  fails_with gcc: "5"
```
I think this might help with the Linux failure.

Reply:
`clickhouse-cpp` requires full support of C++17, which is in GCC starting from version `7`, I guess. How can I explicitly specify the required GCC version to be `>= 7`?

Reply:
We don't have that kind of flexibility, unfortunately. You can change `fails_with` line to `fails_with gcc: "6"` though.